src/SFA.DAS.Campaign.Web/ViewComponents/Hero-Heading/HeroHeadingViewComponent.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Campaign.Web.ViewComponents.Modal;
+
+namespace SFA.DAS.Campaign.Web.ViewComponents
+{
+    public class HeroHeadingViewComponent : ViewComponent
+    {
+        public async Task<IViewComponentResult> InvokeAsync(HeroHeadingType type, string caption, string classes, IHtmlContent content)
+        {
+           
+            return View("Default", new HeroHeadingViewModel(type, caption, classes, content));
+        }
+    }
+
+    public class HeroHeadingViewModel
+    {
+        public HeroHeadingViewModel(HeroHeadingType type, string caption, string classes, IHtmlContent content)
```

Comment:
Should this be in a separate file? Also is it worth changing Caption, Class, Type and Content to be `internal set`

Reply:
moved but cannot be internal set as access is required from  views

Reply:
setters are now changed to internal where appropriate